src/components/OnboardingForm.tsx

```diff
@@ -1,162 +1,389 @@
+import { Input, Label, Select, Form, Title } from "@/shared/components";
+import { genders, educationLevels } from "@/shared/onboarding";
+
 export default function OnboardingForm() {
   return (
     <div className="absolute h-screen w-screen z-10 top-0 left-0 bg-slate-800 overflow-auto">
-      <form className="bg-[#5d417A] w-full md:w-9/12 mx-auto my-10 p-8 rounded-lg">
+      <Form>
         <div className=" border-gray-900/10">
-          <h2 className="text-base font-semibold leading-7 text-white">
-            Personal Information
-          </h2>
-
-          <div className="mt-10 grid grid-cols-1 gap-x-6 gap-y-8 sm:grid-cols-6">
-            <div className="sm:col-span-3">
-              <label
-                className="block text-sm font-medium leading-6 text-white"
-                htmlFor="first-name"
-              >
-                First name
-              </label>
+          <Title> Formulário de Inscrição</Title>
+
+          <div className="mt-10 grid grid-cols-2 gap-x-6 gap-y-8 sm:grid-cols-6">
+            <div className="sm:col-span-2">
+              <Label htmlFor="nome-social">Nome Social</Label>
+              <div className="mt-2">
+                <Input
+                  id="nome-social"
+                  name="nome-social"
+                  type="text"
+                />
+              </div>
+            </div>
+
+            <div className="sm:col-span-2">
+              <Label htmlFor="gender">Gênero</Label>
+              <div className="mt-2">
+                <Select
+                  defaultValue={"selected"}
+                  id="gender"
+                  name="gender"
+                >
+                  {genders.map((gender) => (
+                    <option key={gender} value={gender}>
+                      {gender}
+                    </option>
+                  ))}
+                </Select>
+              </div>
+            </div>
+
+            <div className="sm:col-span-2">
+              <Label htmlFor="idade">Idade</Label>
+              <div className="mt-2">
+                <Input
+                  id="idade"
+                  name="idade"
+                  type="text" //As text
```

Comment:
We need to remove these comments 😆

Reply:
Thanks for the reply Marco! I will be more attentive.